Runtime/Game/Resources/LootLockerConfig.cs

```diff
@@ -144,16 +159,20 @@ public static LootLockerConfig current
         public enum platformType { Android, iOS, Steam, PlayStationNetwork, Unused }
         [HideInInspector]
         public bool developmentMode = true;
-        [HideInInspector]
-        public string url = "https://api.lootlocker.io/game/v1";
-        [HideInInspector]
-        public string adminUrl = "https://api.lootlocker.io/admin";
-        [HideInInspector]
-        public string playerUrl = "https://api.lootlocker.io/player";
-        [HideInInspector]
-        public string userUrl = "https://api.lootlocker.io/game";
-        [HideInInspector]
-        public string baseUrl = "https://api.lootlocker.io";
+
+        [HideInInspector] private static readonly string UrlPrependage = "https://";
```

Comment:
Could call this scheme or protocol, to make it make a little more sense.

Reply:
Thank you, couldn't remember the name

Reply:
Fixed